src/cryptography/x509/certificate_transparency.py

```diff
@@ -44,5 +77,23 @@ def entry_type(self) -> LogEntryType:
         Returns whether this is an SCT for a certificate or pre-certificate.
         """
 
+    @abc.abstractproperty
+    def hash_algorithm(self) -> HashAlgorithm:
```

Comment:
@reaperhulk: Curious for your thoughts, should we just make this one of our hash types, isntead of an enum?

Reply:
SGTM!

Reply:
Changing now!

Reply:
[f66a9e0](https://github.com/pyca/cryptography/pull/7207/commits/f66a9e01c161f98c27b90944c82ce0580194f85e)